mlbench_core/lr_scheduler/pytorch/lr.py

```diff
@@ -342,3 +342,87 @@ def __init__(self, optimizer, alpha):
 
     def f(self, iteration):
         return 1.0 / math.sqrt(max(1, iteration))
+
+
+def perhaps_convert_float(param, total):
```

Comment:
Use a more descriptive name, like `convert_relative_stepsize` or something along those lines so the intention is more clear.